### DISCUSSION TITLE
Got some errors in RustDesk Flutter project

### DISCUSSION THREAD

**Discussion by Mahamudul-Dev** [rustdesk/rustdesk#6367] [General]
Hello experts!
I am trying to run flutter project of RustDesk but I got this error
Can anyone help me how can I solve this?
![Screenshot from 2023-11-10 20-00-33](https://github.com/rustdesk/rustdesk/assets/108647153/13ea405d-98b7-4e40-acb1-d834ef405b36)
![Screenshot from 2023-11-10 19-58-15](https://github.com/rustdesk/rustdesk/assets/108647153/6e9ee822-1a1c-45b7-9579-e21512befee9)

**Comment by TBG-FR** [comment]
Your error message is related to the missing file lib/generated_bridge.dart that should be generated using this script : https://github.com/rustdesk/rustdesk/blob/master/flutter/run.sh

If you face other issues, I gave some other hints here https://github.com/rustdesk/rustdesk/discussions/2535

**Reply by Mahamudul-Dev** [reply]
Okey, Tanks a lot for this information.
